src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,52 @@
+@use "../utils/variables" as *;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: 30px 10px;
+  width: calc($day-size * 7.06);
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: $day-size;
+    height: $day-size;
+    background-color: #eee;
+    box-sizing: border-box;
+
+    border: 1px solid black;
+
+    color: black;
+    font-size: 30px;
+    font-family: Arial, Helvetica, sans-serif;
+
+    transition: transform 0.5s, background-color 0.5s;
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: "#{$i}";
+      }
+    }
+    &:hover {
+      cursor: pointer;
+      transform: translateY(-20px);
+      background-color: #90ee90;
+    }
+  }
+
+  @each $day, $margin in $days {
+    &--start-#{$day} .calendar__day:first-child {
```

Comment:
It doesn't work. You should add `class__element--start-day` to your html file. Checkpoint nr 2.